tests/test_mle.py

```diff
@@ -299,6 +299,38 @@ def test_infer_node_vals_and_errs_networkx(nx_graph_with_reference_mle_instance)
         # assert g_res.nodes[n][node_stddev_prop] == pytest.approx(ref_dg_err)
 
 
+def test_infer_node_vals_and_errs_networkx_multi(nx_graph_with_reference_mle_instance):
```

Comment:
Not necessary, but another testing strategy could be to summarize all parallel edges as a single edge. (Since product of Gaussians is a Gaussian https://github.com/proteneer/timemachine/pull/399#issuecomment-824893476 )